QualityControl/public/common/runModeHeader.js

```diff
@@ -0,0 +1,236 @@
+      ),
+      renderStatusInfoDropdown(filterModel),
+    ]);
+  }
+
+  /**
+   * Renders the status info dropdown
+   * @param {Model} model - root model of the application
+   * @param {FilterModel} filterModel - model that manages filter state
+   * @returns {vnode} - virtual node element
+   */
+  function renderStatusInfoDropdown(filterModel) {
+    return h('.dropdown.mh1', {
+      class: filterModel._statusInfoOpen ? 'dropdown-open' : '',
+    }, [
+      renderInfoButton(filterModel),
+      filterModel._statusInfoOpen && renderDropdownMenu(),
+    ]);
+  }
+
+  /**
+   * Renders the info button
+   * @param {FilterModel} filterModel - model that manages filter state
+   * @returns {vnode} - virtual node element
+   */
+  function renderInfoButton(filterModel) {
+    return h('button.btn.btn-sm', {
+      title: 'Show status information',
+      onclick: (e) => {
+        e.stopPropagation();
+        filterModel._statusInfoOpen = !filterModel._statusInfoOpen;
+        filterModel.notify();
+      },
+      style: 'padding: 2px 6px; font-size: 0.75rem;',
+    }, info());
+  }
+
+  /**
+   * Renders the dropdown menu content
+   * @returns {vnode} - virtual node element
+   */
+  function renderDropdownMenu() {
+    return h('.dropdown-menu.show', [
+      h('.p2.text-left', [
+        h('span.gray-darker', 'Status meanings:'),
+        h('hr'),
+        renderStatusList(),
+      ]),
+    ]);
+  }
+
+  /**
+   * Renders the list of all possible statuses
+   * @returns {vnode} - virtual node element
+   */
+  function renderStatusList() {
+    return h('div', [
+      h('div', ...Object.values(RunStatus).map((status) =>
+        h('label', {
+          style: 'transition: transform 0.2s;',
+          onmouseover: (e) => {
+            e.target.style.transform = 'scale(1.02)';
+            e.target.style.cursor = 'pointer';
+          },
+          onmouseout: (e) => {
+            e.target.style.transform = 'scale(1)';
+            e.target.style.cursor = 'default';
+          },
+        }, h(`span.${getStatusClass(status)}`, { style: 'font-weight:bold' }, status), ' - ', getStatusTitle(status)))),
+    ]);
+  }
+
+  /**
+   * Renders the exit button
+   * @param {FilterModel} filterModel - model that manages filter state
+   * @param {PageModel} pageModel - model that manages the state of the page
+   * @returns {vnode} - virtual node element
+   */
+  function renderExitButton(filterModel, pageModel) {
+    return h('button.btn.btn-sm.btn-outline-secondary', {
+      onclick: async () => await filterModel.deactivateRunsMode(pageModel),
+      title: 'Exit run mode and show all filters',
+    }, 'X Exit');
+  }
+
+  /**
+   * Returns the CSS class based on the run status
+   * @param {string} status - The run status value
+   * @returns {string} - The corresponding CSS class
+   */
+  function getStatusClass(status) {
+    if (!status) {
+      return 'text-muted';
+    }
+
+    switch (status.toLowerCase()) {
+      case 'ended':
```

Comment:
You have the runStatus enum introduced above in the PR. Perhaps you should be using that rather than hardcoded strings?
Moreover, we have status CSS classes declared in the code base already (`bootstrap.css`) such as `success`, `primary`, etc.